sites/fast-creator/app/creator.tsx

```diff
@@ -355,6 +356,24 @@ class Creator extends Editor<{}, CreatorState> {
         }
     };
 
+    private handleWindowMessage = (e: MessageEvent): void => {
+        if (e.data) {
+            try {
+                const messageData = JSON.parse(e.data);
+
+                if (messageData.type === "dataDictionary" && messageData.data) {
+                    this.fastMessageSystem.postMessage({
+                        type: MessageSystemType.initialize,
+                        data: messageData.data,
+                        schemaDictionary,
+                    });
+                }
+            } catch (e) {
+                console.log(e);
```

Comment:
This might be better as a console.error or at least console.warn for better visibility.